.github/workflows/vs16-noexcept-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: VS16-NoExcept-CI
```

Comment:
Not something to solve here, but there *must* be a way for github ci actions to share code ...

Reply:
Yes. But I find it difficult to read and debug complex yaml files. Having distinct tests be in distinct yaml files is easier to me.